pkg/apis/enmasse/v1beta1/types.go

```diff
@@ -289,13 +289,15 @@ type Address struct {
 }
 
 type AddressSpec struct {
-	Address      string            `json:"address"`
-	AddressSpace string            `json:"addressSpace,omitempty"`
-	Type         string            `json:"type"`
-	Plan         string            `json:"plan"`
-	Topic        string            `json:"topic,omitempty"`
-	Subscription *SubscriptionSpec `json:"subscription,omitempty"`
-	Forwarders   []ForwarderSpec   `json:"forwarders,omitempty"`
+	Address           string            `json:"address"`
+	AddressSpace      string            `json:"addressSpace,omitempty"`
+	Type              string            `json:"type"`
+	Plan              string            `json:"plan"`
+	Topic             string            `json:"topic,omitempty"`
+	DeadLetterAddress string            `json:"deadLetterAddress,omitempty"`
+	ExpiryAddress     string            `json:"expiryAddress,omitempty"`
+	Subscription      *SubscriptionSpec `json:"subscription,omitempty"`
+	Forwarders        []ForwarderSpec   `json:"forwarders,omitempty"`
```

Comment:
Missing messageRedalivery?

Reply:
I can add messageRedelivery (and messageTtl) to the specs and status sections of the Go types, however, at the moment nothing is actually consuming them.   ExpiryAddress and DeadLetterAddress are required for the Console, but as the console doesn't let you config plans nor address fields such as ttl/redelivery we don't strictly need them at the minute.

Reply:
Agreed, makes sense to skip those then.